Rolique/UI/Colleagues/ColleaguesTableViewCell.swift

```diff
@@ -37,6 +39,10 @@ class ColleaguesTableViewCell: UITableViewCell {
     containerView.translatesAutoresizingMaskIntoConstraints = false
     containerView.backgroundColor = .white
     
+    userImageView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Use extension method that adds view and set `translatesAutoresizingMaskIntoConstraints` to false right away